### PR TITLE
support to enable follower read in bank/bank2

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What problem does this PR solve? 

Support to enable follower read in bank/bank2.
Port https://github.com/pingcap/schrodinger-test/pull/462 .

### What is changed and how does it work?

Add a new option to enable/disable follower read. And randomly change it when running.

### Check List 

Tests 

 - No code

Code changes

 - Has Go code change

Side effects

Related changes

### Does this PR introduce a user-facing change?:
```release-note
NONE
```

### CODE REVIEW

File: util/util.go [hunk 1]
```diff
@@ -184,3 +185,29 @@ func InstallArchive(ctx context.Context, rawURL string, dest string) error {
...
+
+func setTiDBReplicaRead(job string, db *sql.DB, mode string) {
+	sql := fmt.Sprintf("set @@tidb_replica_read = '%s'", mode)
+	if _, err := db.Exec(sql); err != nil {
+		log.Fatalf("[%s] tidb_replica_read set fail: %v", job, err)
+	}
+}
+
+// RandomlyChangeReplicaRead changes `tidb_replica_read` randomly in a session.
+func RandomlyChangeReplicaRead(job, replicaRead string, db *sql.DB) {
```
Comment (mahjonp) [inline util/util.go]: IMO, Putting these three methods here isn't suitable.

Comment (hicqu) [reply util/util.go]: They are not only for bank/bank2. All other cases can use them.

Comment (mahjonp) [reply util/util.go]: OK, but "bank2" is dead coded here. :)

Comment (hicqu) [reply util/util.go]: sorry, don't see it.

**Comment by mahjonp** [comment]
please run `make tidy` to fix the CI.

**Pull request merged**